nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/BaseHealthCheck.java

```diff
@@ -285,7 +286,7 @@ public void sendNotification(String name, HealthCheckResult checkResult, Level l
         String subject = "Health Check notification with severity level: " + level.getName();
         String messageFormatted = getMessageFormatted(new Object[]{name, getCumulativeMessages(checkResult.getEntries())});
 
-        List<String> enabledNotifiers = healthCheckService.getEnabledNotifiers();
+        Collection<String> enabledNotifiers = healthCheckService.getEnabledNotifiers();
```

Comment:
Just noticed that factory in line below is also still on. Is there a reason to not have the `newBuilder` be a static method in the builder class?

Reply:
The factory is needed to inject the HK2 properties required in the notification. I've reworked the builder API so it should be more clear now!